frontends/web/src/routes/account/walletconnect/dashboard.tsx

```diff
@@ -106,14 +106,17 @@ export const DashboardWalletConnect = ({ code, accounts }: TProps) => {
               {hasSession &&
             <div className={styles.sessionCardsContainer}>
               <p className={styles.allSessionsHeading}>{t('walletConnect.dashboard.allSessions')}</p>
-              {sessions.map(session =>
-                <WCSessionCard
-                  key={session.topic}
-                  receiveAddress={getAddressFromEIPString(session.namespaces['eip155'].accounts[0])}
-                  metadata={session.peer.metadata}
-                  onDisconnect={() => handleDisconnectSession(session.topic)}
-                />
-              )}
+              {sessions.map(session => {
+                console.log('session: ', session);
```

Comment:
remove console.log?

Reply:
damn, old commit, swear I removed that :)